wagtail_content_import/mappers/tests.py

```diff
@@ -0,0 +1,58 @@
+from django.contrib.auth.models import User
+from django.test import TestCase
+
+from wagtail.images import get_image_model
+
+from unittest.mock import MagicMock
+
+from .converters import ImageConverter, RichTextConverter, TextConverter, TableConverter
+from ..parsers.tables import Table, Cell
+
+
+class testConverters(TestCase):
```

Comment:
Class names should start with a capital letter

Reply:
Thanks for spotting!